common/src/main/java/com/sequenceiq/cloudbreak/telemetry/databus/AbstractDatabusStreamConfiguration.java

```diff
@@ -29,6 +31,18 @@ public String getDbusStreamName() {
         return dbusStreamName;
     }
 
+    public boolean isStreamingEnabled() {
```

Comment:
these could be moved in the interface as default implementation or just not implement it and throw an exception or something similar, by the way, here the default should be false